Day3/src/style.scss

```diff
@@ -0,0 +1,48 @@
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+  }
+  
+  $background-color_1: grey;
+
+  .header {
+      display: flex;
+      width: 100%;
+      height: 7vh;
+      justify-content: space-between;
+      align-items: center;
+      padding: 40px;
+      background-color: $background-color_1;
+      .h-logo {
```

Comment:
Can name this as `h-logo` class as `header-logo` and can use as `&-logo` instead.